site/pages/create-disruption.tsx

```diff
@@ -10,58 +11,195 @@ interface CreateDisruptionProps {
     inputs: DisruptionInfo;
 }
 
+enum Reason {
+    roadWorks = "Road Works",
+    vandalism = "Vandalism",
+    routeDiversion = "Route Diversion",
+    specialEvent = "Special Event",
+}
+
+const reasons: {
```

Comment:
Should we move these into the constants file?

Reply:
the variable is now removed and not required

Reply:
Unsure what you mean @alvak1  - what variable?

Reply:
Apologies, mistook this variable to the another variable that was deleted. I have moved it to the constants file